backend/Node/routes/spotifyRoute.ts

```diff
@@ -87,44 +84,71 @@ router.get('/getMe/:access_token', async (req: any, res: any) => {
 });
 
 // get user top tracks
+router.get('/getMyTopTracks/:access_token', async (req: any, res: any) => {
+	try {
+		if (!req.params.access_token)
+			return res.status(400).send('failed to authenticate');
+
+		const url = 'https://api.spotify.com/v1/me/top/tracks';
+
+		const topTracks: any = await axios.get(url, {
+			headers: { Authorization: `Bearer ${req.params.access_token}` },
+		});
+
+		if (topTracks) {
+			let topTracksIds = topTracks.data.items;
+			topTracksIds = topTracksIds.map((track: any) => track.id);
```

Comment:
Same here, can we just map the `topTrackIds` directly and type `topTrackIds` as `string[]`?

Reply:
implemented